### ISSUE TITLE
[Bug Report][3.0.0] v-btn is unable to set a "medium" radius when the default has been changed (blueprint md3 sets `xl`)

### ISSUE THREAD

**Issue by mhio** [vuetifyjs/vuetify#16049]
### Environment
**Vuetify Version:** 3.0.0
**Vue Version:** 3.2.41
**Browsers:** Chrome 106.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
- Set `import { md3 } from 'vuetify/blueprints'` which results in a default pill shaped button
- Add `v-btn` component without a `rounded` value. 
- Add `v-btn` component with a `rounded` value `sm`. 
- Add `v-btn` component with a `rounded` value `lg`. 

### Expected Behavior
button `rounded` can be set between `sm` and `lg`

### Actual Behavior
The default button `rounded` value is not between `md` and `lg` 

### Reproduction Link
[https://codepen.io/mio8o8/pen/GRGrowW](https://codepen.io/mio8o8/pen/GRGrowW)

### Other comments
Maybe needs a `md`/`.rounded-md` to also imply what the default vuetify value rounded value is. 
Looking at the [rounded composable](https://github.com/vuetifyjs/vuetify/blob/0a562f087658c1e533a4f431b70031daa4bcde01/packages/vuetify/src/composables/rounded.ts#L35-L40) led me to testing `true` values which ended up working in the codepen for `rounded="true", but setting the same "true" does not work (or apply) on my local vite build. 

On the flip side, Pagination buttons do use the "medium" value under this setup. Maybe that is also a bug, by not respecting the blueprint default.

**Comment by KaelWD** [comment]
I don't understand what the problem is here.

**Comment by mhio** [comment]
I can't apply the regular rounded radius (between `sm` and `lg`) to a button when using the `md3` blueprint. 

The API on VButton assumes no `rounded` prop is the way to set the "regular radius" (between `sm` and `lg`) but the default button style is different in md3.

**Comment by mhio** [comment]
Sorry, I think the reason I can't get that 4px border radius is related to using the `icon` prop on the button rather than the blueprint.

**Comment by mhio** [comment]
Both md3 and buttons with an icon exhibit the inconsistency in the progression from rounded 0 -> xl
```
      <div class="d-flex ma-1">
        <v-btn icon="$menu" variant="outlined" class="mr-1" width="4rem" rounded="0"/>
        <v-btn icon="$menu" variant="outlined" class="mr-1" width="4rem" rounded="sm"/>
        <v-btn icon="$menu" variant="outlined" class="mr-1" width="4rem" />
        <v-btn icon="$menu" variant="outlined" class="mr-1" width="4rem" rounded="lg"/>
        <v-btn icon="$menu" variant="outlined" class="mr-1" width="4rem" rounded="xl"/>
      </div>
      <div class="d-flex ma-1">
        <v-btn variant="outlined" class="mr-1" rounded="0"/>
        <v-btn variant="outlined" class="mr-1" rounded="sm"/>
        <v-btn variant="outlined" class="mr-1" />
        <v-btn variant="outlined" class="mr-1" rounded="lg"/>
        <v-btn variant="outlined" class="mr-1" rounded="xl"/>
      </div>
```
md2:
<img width="351" alt="md3 buttons" src="https://user-images.githubusercontent.com/37205532/200736675-1a5fee82-06b3-462a-a0e9-b7a869f59fb3.png">

md3:
<img width="353" alt="md3 buttons" src="https://user-images.githubusercontent.com/37205532/200736772-5f435837-a312-4514-89a1-5a31d5661876.png">



So an explicit `rounded="md"` is what I was expecting to be able to set.

**Comment by johnleider** [comment]
fixed by bb8cdc208f59835ca680765def2c8730b2b4abeb
